### PR TITLE
bump pip version to ensure we can install vsphere SDK

### PR DESCRIPTION
**Pull request by goneri**
Vsphere automation SDK cannot be installed with pip 19.0.2:

```
(...)
02:27 Collecting git+https://github.com/vmware/vsphere-automation-sdk-python.git (from -r /root/ansible/test/lib/ansible_test/_data/requirements/integration.cloud.vcenter.txt (line 2))
02:27   Cloning https://github.com/vmware/vsphere-automation-sdk-python.git to /tmp/pip-req-build-pm27t16b
02:33 Requirement already satisfied: pyvmomi in /usr/local/lib/python3.6/dist-packages (from -r /root/ansible/test/lib/ansible_test/_data/requirements/integration.cloud.vcenter.txt (line 1)) (6.7.1.2018.12)
02:33 Requirement already satisfied: lxml>=4.3.0 in /usr/local/lib/python3.6/dist-packages (from vSphere-Automation-SDK==1.4.0->-r /root/ansible/test/lib/ansible_test/_data/requirements/integration.cloud.vcenter.txt (line 2)) (4.4.0)
02:33 Processing ./\\localhost/tmp/pip-req-build-pm27t16b/lib/vapi-runtime/vapi_runtime-2.12.0-py2.py3-none-any.whl
02:33 Could not install packages due to an EnvironmentError: [Errno 2] No such file or directory: '/root/ansible/\\\\localhost/tmp/pip-req-build-pm27t16b/lib/vapi-runtime/vapi_runtime-2.12.0-py2.py3-none-any.whl'
```

See: https://github.com/ansible/ansible/pull/62022

### CODE REVIEW

File: requirements/integration.cloud.vcenter.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 pyvmomi
+git+https://github.com/vmware/vsphere-automation-sdk-python.git
```
Comment (mattclay) [inline requirements/integration.cloud.vcenter.txt]: These changes should be made in ansible/ansible first, and then updated by running `make update && make freeze` instead of manually editing them here.

File: files/requirements.sh [hunk 2]
```diff
@@ -52,7 +52,7 @@ fi
 echo "Using constraints file: ${constraints}"
 
 get_pip_tmp="/tmp/get-pip.py"
-pip_version="19.0.2"
+pip_version="19.2.3"
```
Comment (mattclay) [inline files/requirements.sh]: What happens if the SDK is installed using pip 9.0.3, which is installed for Python 2.6?

Comment (goneri) [reply files/requirements.sh]: The SDK don't support Python 2.6: https://github.com/vmware/vsphere-automation-sdk-python#abstract
It's the same for pyVmomi.

File: requirements/integration.cloud.vcenter.txt (hunk 1 above)
Comment (goneri) [reply requirements/integration.cloud.vcenter.txt]: ok, good to know.

File: freeze/2.7.txt [hunk 3]
```diff
@@ -67,7 +67,7 @@ funcsigs==1.0.2
 future==0.17.1
 futures==3.3.0
 google-auth==1.6.3
-hcloud==1.4.0
+hcloud==1.4.1
```
Comment (mattclay) [inline freeze/2.7.txt]: The hcloud changes should be done in a separate PR. Also, instead of manually updating the freeze files, go through the `make update && make freeze` process. I recommend doing so after the changes in this PR are merged.

File: files/requirements.sh (hunk 2 above)
Comment (mattclay) [reply files/requirements.sh]: Do we know why the newer version of pip is required to install the SDK? Is it working around a bug in the SDK, or is it a legitimate bug in pip that was fixed?

Comment (mattclay) [reply files/requirements.sh]: It looks like that bug fix was included in pip 19.1.
